edx_proctoring/static/proctoring/templates/student-proctored-exam-attempts-grouped.underscore

```diff
@@ -149,9 +150,22 @@
                     </tr>
                 </thead>
                 <% if (is_proctored_attempts) { %>
-                <tbody>
                     <% _.each(proctored_exam_attempts, function(proctored_exam_attempt, dashboard_index){ %>
+                    <tbody
```

Comment:
Because HTML table structures have strict parent/child definitions, I've modified the layout of the table to include multiple tbodys. This allows us to better group attempts and apply styling to those groups.